.github/workflows/tf-plan.yml

```diff
@@ -0,0 +1,69 @@
+name: "Terraform Plan"
+
+# Trigger when a pull request is received
+on:
+  pull_request:
+
+jobs:
+  terraform-plan:
+    name: "Terraform Plan"
+    runs-on: ubuntu-latest
+    steps:
+      # Checkout the code
+      # Marketplace: https://github.com/marketplace/actions/checkout
+      - name: "Setup - Checkout"
+        uses: actions/checkout@v2.1.0
+       
+      # Static analysis of Terraform templates to spot potential security issues
+      # Marketplace: https://github.com/marketplace/actions/terraform-security-scan
+    #  - name: "Setup - Security Scan"
+    #    uses: triat/terraform-security-scan@v1
+
+
+      # Provides AWS credentials to Terraform
+      # By default, Terraform checks the home directory for a .aws folder with a credential file
+      # Documentation: https://docs.aws.amazon.com/ses/latest/DeveloperGuide/create-shared-credentials-file.html
+      - name: "Setup - Build AWS Credentials"
+        run: |
+          mkdir -p ~/.aws
+          echo "[default]" > ~/.aws/credentials
+          echo "aws_access_key_id = ${{ secrets.AWS_ACCESS_KEY_ID }}" >> ~/.aws/credentials
+          echo "aws_secret_access_key = ${{ secrets.AWS_SECRET_ACCESS_KEY }}" >> ~/.aws/credentials
+                
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}   
+          
+      
+         
+        #Init pulls credentials from the .aws/credentials file by default
+      - name: "Run - Terraform plan"
+        uses: hashicorp/setup-terraform@v1.0.1
+
+      - name: "Terraform plan"
+        id: plan
+        run: |
+          mkdir -p ~/.ssh
```

Comment:
Do we need this ssh now as we have removed github for RDS?

Reply:
need to check without .ssh